### ISSUE TITLE
Aesthetic: Responsive Optimization - Transform Create Modal into full View

### ISSUE THREAD

**Issue by jaeko44** [megamsys/nilavu#1067]
In order to be able to optimize the create VM/app process for phones. I need a seperate view for devices under a certain width to use a full page view instead of a modal.

Meaning we don't popup a modal and instead redirect them to a Create VM page with the same inner contents as the modal.

This is because modals don't pair nicely with phones & are unnecessary.

**Comment by jaeko44** [comment]
`
detectmob() { 
  if( navigator.userAgent.match(/Android/i)
  || navigator.userAgent.match(/webOS/i)
  || navigator.userAgent.match(/iPhone/i)
  || navigator.userAgent.match(/iPad/i)
  || navigator.userAgent.match(/iPod/i)
  || navigator.userAgent.match(/BlackBerry/i)
  || navigator.userAgent.match(/Windows Phone/i)
  ){
      return true;
    }
  else if (window.innerWidth <= 800 && window.innerHeight <= 600) {
    return true;
  }
  else {
      return false;
    }
  }
`

The above is a JS snippet I've previously used in a different project to detect if the user is using a phone.

**Comment by vinomca-megam** [comment]
This is system based only.we will fix it in next release.

**Comment by vinomca-megam** [comment]
What does you mean full page view? is it like separate page instead of Modal?

**Comment by jaeko44** [comment]
Yes - but this is very low priority. Don't worry about it till after 1.5 release.

**Comment by rajthilakmca** [comment]
@jaeko44 if we separate the create VM page instead of modal, then the maximum number of issues are closed, so could i replace it ?

**Comment by jaeko44** [comment]
ok, we can replace using a modal till we fix the issues in the future. That is fine, we can use a full page.

**Comment by rajthilakmca** [comment]
ok fine, i'll replace to page
